### DISCUSSION TITLE
New Contributors section missing after generating changelog

### DISCUSSION THREAD

**Discussion by pplmx** [orhun/git-cliff#830] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the bug

New Contributors section missing after generating changelog

### Steps To Reproduce

```bash
git clone https://github.com/x-pt/template.git
cd template
git cliff -o CHANGELOG.md

# see the deletion lines
git diff CHANGELOG.md
```

### Expected behavior

Existed `New Contributor` section should be kept.

### Screenshots / Logs

![image](https://github.com/user-attachments/assets/c6c4b997-a840-4dbe-acec-fc084ca1755f)

### Software information

- Operating system: macOS
- Rust version: rustc 1.80.1 (3f5fd8dd4 2024-08-06)
- Project version: git-cliff 2.5.0

**Comment by orhun** [comment, marked as answer]
It looks like in commit `abafc23ee8f236c68948570a33d2f92b11befd2d` you started skipping the dependency bumps by dependabot and that's why they no longer appear in the changelog:

```diff
- ## New Contributors ❤️

- * @github-actions[bot] made their first contribution
```

**Reply by pplmx** [reply]
Oh, my bad. Thanks a lot! :)

**Comment by orhun** [comment]
Btw I found a bug which might or might be related to this: #891
